common/presenters/table/object-to-table-head.js

```diff
@@ -1,8 +1,11 @@
 const i18n = require('../../../config/i18n')
 
 function objectToTableHead(schemaEl) {
+  const { head } = schemaEl
+  const prop = head.html ? 'html' : 'text'
```

Comment:
I think this can be improved a little to match the [`object-to-table-row`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/53987ba7e66e6ef1e8b57065a7c62e8cd872ae15/common/presenters/table/object-to-table-row.js#L17-L21) solution as currently if neither `html` or `text` are actually passed to this function it will still output an empty `text` property.

Reply:
This change still produces the same error where if neither `html` or `text` properties are passed, a `text` property is still output with an empty string.

Reply:
I will add `pickBy` but to be honest, I don't quite see a scenario in which this matters. Maybe you can help me figure out one?